pkg/splunk/client/awss3client.go

```diff
@@ -90,3 +104,13 @@ func (awsclient *AWSS3Client) GetAppsList() (S3Response, error) {
 
 	return s3Resp, nil
 }
+
+// GetInitContainerImage returns the initContainer image to be used with this s3 client
+func (awsclient *AWSS3Client) GetInitContainerImage() string {
+	return ("amazon/aws-cli")
+}
+
+// GetInitContainerCmd returns the init container command on a per app source basis to be used by the initContainer
+func (awsclient *AWSS3Client) GetInitContainerCmd(endpoint string, bucket string, path string, appSrcName string, appMnt string) []string {
+	return ([]string{fmt.Sprintf("--endpoint-url=%s", endpoint), "s3", "sync", fmt.Sprintf("s3://%s/%s", bucket, path), fmt.Sprintf("%s/%s", appMnt, appSrcName) /*, "--include \"*.tgz\"", "--include \"*.spl\""*/})
```

Comment:
Nit: Do you wish to keep the last commented part?

Reply:
Removed.